deploy/manifests/service-monitor.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
we should add the ServiceMonitor manifest here since that would add a dependency on prometheus-operator

Reply:
Or could we delete this ServiceMonitor, to remove the dependency to prometheus-operator?

Technically, prometheus-operator is not required to use prometheus-adapter.

Reply:
Or could we delete this ServiceMonitor, to remove the dependency to prometheus-operator?

Technically, prometheus-operator is not required to use prometheus-adapter.

Reply:
I meant shouldn't, the idea is to remove it to avoid having an unnecessary dependency yes.